include/zephyr/dt-bindings/pinctrl/renesas/pinctrl-r8a77951.h

```diff
@@ -72,38 +72,38 @@
 #define PIN_AVB_AVTP_CAPTURE_A     RCAR_GP_PIN(2, 14)
 #define PIN_SD0_CLK                RCAR_GP_PIN(3, 0)
 #define PIN_SD0_CMD                RCAR_GP_PIN(3, 1)
-#define PIN_SD0_DATA0              RCAR_GP_PIN(3, 2)
-#define PIN_SD0_DATA1              RCAR_GP_PIN(3, 3)
-#define PIN_SD0_DATA2              RCAR_GP_PIN(3, 4)
-#define PIN_SD0_DATA3              RCAR_GP_PIN(3, 5)
+#define PIN_SD0_DAT0               RCAR_GP_PIN(3, 2)
+#define PIN_SD0_DAT1               RCAR_GP_PIN(3, 3)
+#define PIN_SD0_DAT2               RCAR_GP_PIN(3, 4)
+#define PIN_SD0_DAT3               RCAR_GP_PIN(3, 5)
 #define PIN_SD1_CLK                RCAR_GP_PIN(3, 6)
 #define PIN_SD1_CMD                RCAR_GP_PIN(3, 7)
-#define PIN_SD1_DATA0              RCAR_GP_PIN(3, 8)
-#define PIN_SD1_DATA1              RCAR_GP_PIN(3, 9)
-#define PIN_SD1_DATA2              RCAR_GP_PIN(3, 10)
-#define PIN_SD1_DATA3              RCAR_GP_PIN(3, 11)
+#define PIN_SD1_DAT0               RCAR_GP_PIN(3, 8)
+#define PIN_SD1_DAT1               RCAR_GP_PIN(3, 9)
+#define PIN_SD1_DAT2               RCAR_GP_PIN(3, 10)
+#define PIN_SD1_DAT3               RCAR_GP_PIN(3, 11)
 #define PIN_SD0_CD                 RCAR_GP_PIN(3, 12)
 #define PIN_SD0_WP                 RCAR_GP_PIN(3, 13)
 #define PIN_SD1_CD                 RCAR_GP_PIN(3, 14)
 #define PIN_SD1_WP                 RCAR_GP_PIN(3, 15)
 #define PIN_SD2_CLK                RCAR_GP_PIN(4, 0)
-#define PIN_SD2_CMD                RCAR_GP_PIN(4,  1)
-#define PIN_SD2_DATA0              RCAR_GP_PIN(4,  2)
-#define PIN_SD2_DATA1              RCAR_GP_PIN(4,  3)
-#define PIN_SD2_DATA2              RCAR_GP_PIN(4,  4)
-#define PIN_SD2_DATA3              RCAR_GP_PIN(4,  5)
-#define PIN_SD2_DS                 RCAR_GP_PIN(4,  6)
-#define PIN_SD3_CLK                RCAR_GP_PIN(4,  7)
-#define PIN_SD3_CMD                RCAR_GP_PIN(4,  8)
-#define PIN_SD3_DATA0              RCAR_GP_PIN(4,  9)
-#define PIN_SD3_DATA1              RCAR_GP_PIN(4,  10)
-#define PIN_SD3_DATA2              RCAR_GP_PIN(4,  11)
-#define PIN_SD3_DATA3              RCAR_GP_PIN(4,  12)
-#define PIN_SD3_DATA4              RCAR_GP_PIN(4,  13)
-#define PIN_SD3_DATA5              RCAR_GP_PIN(4,  14)
-#define PIN_SD3_DATA6              RCAR_GP_PIN(4,  15)
-#define PIN_SD3_DATA7              RCAR_GP_PIN(4,  16)
-#define PIN_SD3_DS                 RCAR_GP_PIN(4,  17)
+#define PIN_SD2_CMD                RCAR_GP_PIN(4, 1)
```

Comment:
I see no reason for these changes, and they create new conflicts with an already existing PR: https://github.com/zephyrproject-rtos/zephyr/pull/60285.

Could you please wait until the mentioned PR is merged and make these changes in all the necessary places, or alternatively, abandon these modifications?

Reply:
Hmm, sorry but I can't see the conflict, #60285 is not editing file under `include/zephyr/dt-bindings/pinctrl/renesas`.
Am I wrong ?

I was just taking advantage of this PR to harmonize the file, but they can be removed.

Reply:
> Am I wrong ?

No, of course. I'm just trying to say that after your changes, I'll have a little bit more pain in my life :face_exhaling:, because it will be needed to update my PR in which I've [referenced the names that you changed](https://github.com/zephyrproject-rtos/zephyr/pull/60285/files#diff-4f51a3fd64a2a3c77aa6b5c17cc4d1d11d7548095c2a3a12d68d3be1a933d762R27). It will increase the review even more because, after the update, I should gather ack(s) again, and experience says that it takes at least 2 weeks.

Of course, I'm not able to mark this PR with NW because of this reason. This is just a gentle request.

Reply:
Sorry, I hadn't noticed you used this pin in the board pinctrl file.
I can add DNM to my PR waiting for your's (that is indeed harder to merge) to be merged !